src/js_realm_object.hpp

```diff
@@ -295,6 +297,22 @@ std::vector<String<T>> RealmObjectClass<T>::get_property_names(ContextType ctx,
     return names;
 }
 
+template <typename T>
```

Comment:
Would be nice to have a comment here explaining the arguments.
For example: https://github.com/realm/realm-js/blob/master/src/js_realm.hpp#L827
We have decided as a team that any new functions should come with comment block headers.